Update templates.rst

andrijapanic opened the pull request:
Ignore this one...

pdion891 commented:
Hi Andrija, do you know a reason why we should keep .in on the filename, so far, the cloud-set-guest-password and cloud-set-guest-sshkey script are always rename to remove the .in which I suspect as been added to avoid download problem since they are bash script.

Thanks

andrijapanic commented:
Hi Pierrre - actually I dont see any value adding .in extension - but the rename part was missing - so the docs is "incomplete" - we can as well just add 1 line with "mv cloud-set-guest-password.in cloud-set-guest-password" - it just wasnt complete, so I changed it that way...
I personaly did rename it, removed .in extension...

pdion891 commented:
Hi Andrija, I cannot apply to patch, I've made a lot of change yesturday which confict :-S. Do you think you could rebase your fork and resubmit a new patch?

Thanks,
